schema-tool/README.md

```diff
@@ -0,0 +1,186 @@
+# Schema Tool for Scalar DB
+This tool creates Scalar DB schemas on Cosmos DB, DynamoDB, Cassandra and a JDBC database.
+  - For Cosmos DB, this tool creates databases(collections) and tables(containers), also inserts metadata which is required by Scalar DB.
+  - For DynamoDB, this tool creates tables named with the database names and table names, also inserts metadata which is required by Scalar DB.
+  - For Cassandra, this tool creates databases(keyspaces) and tables. You can specify the compaction strategy, the network topology strategy, and the replication factor as well.
+  - For a JDBC database, this tool creates databases(schemas, except Oracle) and tables, also inserts metadata which is required by Scalar DB.
+  - You don't have to add Scalar DB metadata for transactions. This tool automatically adds them when you set the `transaction` parameter `true` in your schema file.
```

Comment:
Maybe you just copied it from the existing but I think this should be explained later in an appropriate place.